addons/pointwatch/FIXING POINTWATCH.txt

```diff
@@ -17,6 +17,7 @@ Method 1 - POLUtils
 5) Wait for it to load, then export the entire table as xml
 6) Open the xml file up in a text editor, like Notepad or Notepad++.
 7) Control-F for the key phrase.
+ --look for: ≺Set Color #5≻Visitant Light Intensity≺Set Color #1≻
```

Comment:
Would remove that, it's a very specific example, and the `<Set Color #x>` stuff will probably only confuse people.